hi-ml-histopathology/src/histopathology/datasets/base_dataset.py

```diff
@@ -61,6 +62,7 @@ def __init__(self,
         from the dataset CSV file, e.g. after some filtering. If given, overrides `dataset_csv`.
         :param train: If `True`, loads only the training split (resp. `False` for test split). By
         default (`None`), loads the entire dataset as-is.
+        :param validate_columns: Whether to call `validate_columns()` at the end of `__init__()`.
```

Comment:
It is similar to the param in `SlidesDataset` which was already implemented. It is just to call `validate_columns()` if true - the code in this method was earlier in `init()` of `TilesDataset` but now just moved - this is also similar to `SlidesDataset` which has `validate_columns()`. I have used it to change the dataframe at runtime so that the columns correspond to parent `TilesDataset`, see `PandaTilesDataset`

Reply:
Thank you, I have now explained this in the docstring